### ISSUE TITLE
After upgrade from 0.35.2 to 0.36.2 GraqhQL comment does not work

### ISSUE THREAD

**Issue by wongyouth** [redwoodjs/redwood#3313]
Add a comment line for field in file `api/src/graphql/userExamples.sdl.js`

```
  input UpdateUserExampleInput {
    email: String
    """
    Name is a nickname
    """
    name: String
  }
```

will raise error

```
api | (node:1723) UnhandledPromiseRejectionWarning: GraphQLError: Syntax Error: Expected Name, found BlockString "Name is a nickname".
api |     at syntaxError (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/error/syntaxError.js:15:10)
api |     at Parser.expectToken (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:1413:40)
api |     at Parser.parseName (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:98:22)
api |     at Parser.parseInputValueDef (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:911:21)
api |     at Parser.optionalMany (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:1503:28)
api |     at Parser.parseInputFieldsDefinition (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:1065:17)
api |     at Parser.parseInputObjectTypeDefinition (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:1049:23)
api |     at Parser.parseTypeSystemDefinition (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:720:23)
api |     at Parser.parseDefinition (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:150:23)
api |     at Parser.many (/Users/ryan/code/demos/redwood-v0.36.2/node_modules/graphql/language/parser.js:1523:26)
```

**Comment by dac09** [comment]
Thanks for reporting this @wongyouth and a great specific bug report - we've seen this happening due to the latest version of graphql-tools. We will issue a patch as soon as its fixed on their side!

**Comment by Tobbe** [comment]
@dac09 Are the graphql-tools guys aware of the issue? I took a quick look through their issues, but didn't spot anything. https://github.com/ardatan/graphql-tools/issues

**Comment by Tobbe** [comment]
Found it :) https://github.com/ardatan/graphql-tools/issues/3244
And it's fixed! Even better :)

**Comment by thedavidprice** [comment]
Hi all, this will be resolved via #3371, which is targeted for v0.37 release.
